tests/ui_tests/app/helpers/locator.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from selenium.webdriver.common.by import By
+
+
+@dataclass
+class Locator:
+    """Describes a locator on a webpage"""
+
+    by: By
+    path: str = ""
```

Comment:
It seems to me, `path` is related to `xpath` search strategy. But we have not only `xpath`. I think need more universality for the field name

Reply:
like value?

Reply:
Actually, I don't have any good idea :) Mb `locator_value` ? 
@dgusakov what do you think?

Reply:
navigation/locating value? definition? identificator?

Reply:
changed to value